pallets/parachain-staking/src/inflation.rs

```diff
@@ -82,10 +82,32 @@ pub fn perbill_annual_to_perbill_round(
 		max: annual_to_round(annual.max),
 	}
 }
+
+/// Convert an annual inflation rate to a per-round inflation rate without considering compounding.
+/// The calculation is simply dividing the annual rate by the number of rounds per year.
+pub fn perbill_annual_to_perbill_round_simple(
+	annual: Range<Perbill>,
+	rounds_per_year: u32,
+) -> Range<Perbill> {
+	let annual_to_round_simple = |annual: Perbill| -> Perbill {
+		// Convert the annual rate from Perbill to a fractional representation.
+		let x = I64F64::from_num(annual.deconstruct()) / I64F64::from_num(Perbill::ACCURACY);
+		// Divide the annual rate by the number of rounds to get the per-round rate.
+		let y: I64F64 = x / I64F64::from_num(rounds_per_year);
+		// Convert back to Perbill, rounding as necessary.
+		Perbill::from_parts((y * I64F64::from_num(Perbill::ACCURACY)).floor().to_num::<u32>())
+	};
+	Range {
+		min: annual_to_round_simple(annual.min),
+		ideal: annual_to_round_simple(annual.ideal),
+		max: annual_to_round_simple(annual.max),
+	}
+}
+
 /// Convert annual inflation rate range to round inflation range
 pub fn annual_to_round<T: Config>(annual: Range<Perbill>) -> Range<Perbill> {
 	let periods = rounds_per_year::<T>();
-	perbill_annual_to_perbill_round(annual, periods)
```

Comment:
this function is used somewhere else as well, in `InflationInfo::reset_round()`. Shall we replace that usage as well?

Reply:
one last thing, since the other function is now unused, maybe denote the name with underscore:

```rs
fn _perbill_annual_to_perbill_round(
```

Reply:
there's not prob: as it is a pub fuction (wrong but that's the original code) there's no warning. 
In case we refactor the pub modifier we'll address it.